spec/services/user_signup_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe UserSignup do
+  describe "#sign_up" do
+    let(:valid_user_info) { User.new(
```

Comment:
This is more than user info, I would name it `valid_user` to be more accurate since it is indeed a `User`. The same goes for `invalid_user_info` below.